src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -24,22 +28,23 @@ export const styles = StyleSheet.create({
     flexDirection: 'column',
     height: height - 85
   },
+  flatlistContainer: {
+    paddingTop: 5,
+    paddingBottom: 20
+  },
   chatContainer: {
     display: 'flex',
     height: '100%'
   },
   inputContainer: {
     backgroundColor: colors.white,
-    position: 'absolute',
-    bottom: 0,
-    // height: 50,
-    left: 0,
-    right: 0,
     zIndex: 100,
     padding: 8,
-    paddingBottom: 16,
     borderTopColor: colors.lightgrey,
-    borderTopWidth: 1
+    borderTopWidth: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the styles seem to be focused on improving the layout and positioning of elements. However, it's important to note that the `position: 'absolute'` style was removed from the `inputContainer`. This could potentially affect the positioning of this container in relation to its parent component. If the intention was to make the input container always visible at the bottom of the screen, you might want to reconsider this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->